Running-Locally.md

```diff
@@ -0,0 +1,88 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Running The Demo Raincatcher Solution Locally](#running-the-demo-raincatcher-solution-locally)
+  - [Overview](#overview)
+  - [Requirements](#requirements)
+    - [MongoDB](#mongodb)
+    - [Ruby (Optional)](#ruby-optional)
+  - [Clone All Of The Raincatcher Modules and Demo Apps](#clone-all-of-the-raincatcher-modules-and-demo-apps)
+  - [Notes On Running Locally](#notes-on-running-locally)
+    - [Using Drag & Drop Apps Locally](#using-drag-&-drop-apps-locally)
+    - [Using A Local Service To Authenticate Users](#using-a-local-service-to-authenticate-users)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Running The Demo Raincatcher Solution Locally
+
+## Overview
+
+This guide presents the steps necessary to:
+
+- Run the Raincatcher demo solution locally.
+- Edit Raincatcher modules and have the results reflected in the demo solution.
+
+## Requirements
+
+There are some requirements to getting up and running locally with Raincatcher
+
+### MongoDB
+
+The *raincatcher-demo-cloud* app and *raincatcher-demo-auth* app requires a running MongoDB instance running. See the [MongoDB Installation Guide](https://docs.mongodb.com/manual/installation/) for details on how to install MongoDB.
+
+### Ruby (Optional)
+
+If you want to use the [Raincatcher CLI Tool](https://github.com/feedhenry-raincatcher/raincatcher-cli) to manage starting all of the demo apps, then you will need to install [Ruby](https://www.ruby-lang.org/en/documentation/installation/).
+
+
+## Clone All Of The Raincatcher Modules and Demo Apps
+
+The [Raincatcher CLI Tool](https://github.com/feedhenry-raincatcher/raincatcher-cli) is a useful tool for getting set up with all of the Raincatcher modules and Demo Apps. Follow the steps in the README file to get set up for local development.
+
+## Notes On Running Locally
+
+Some Raincatcher modules require features in the Red Hat Mobile Application Platform (RHMAP) and therefore require access to the RHMAP platform.
+
+### Using Drag & Drop Apps Locally
+
+The *raincatcher-appforms* module requires that the containing cloud app has access to a running MBaaS. This allows the local *raincatcher-demo-cloud* app to access
```

Comment:
It might be important to point out that `MBaaS` here is a term in the RHMAP context